doc/visual-programming/source/widgets/data/continuize.md

```diff
@@ -1,33 +1,46 @@
 Continuize
 ==========
 
-Turns discrete attributes into continuous dummy variables.
+Turns discrete variables (attributes) into numeric ("continuous") dummy variables.
 
 **Inputs**
 
 - Data: input data set
 
 **Outputs**
 
-- Data: data set with continuized instances
+- Data: transformed data set
 
-The **Continuize** widget receives a data set in the input and outputs the same data set in which the discrete attributes (including binary attributes) are replaced with continuous ones.
+The **Continuize** widget receives a data set in the input and outputs the same data set in which the discrete variables (including binary variables) are replaced with continuous ones.
 
 ![](images/Continuize-stamped.png)
 
-1. [Continuization methods](https://en.wikipedia.org/wiki/Continuity_correction), which define the treatment of multivalued discrete attributes. Say that we have a discrete attribute status with the values low, middle and high, listed in that order. Options for their transformation are:
+1. Define the treatment of non-binary categorical variables.
 
-   - **First value as base**: the attribute will be transformed into two continuous attributes, status=middle with values 0 or 1 signifying whether the original attribute had value middle on a particular example, and similarly, status=high. Hence, a three-valued attribute is transformed into two continuous attributes, corresponding to all except the first value of the attribute.
-   - **Most frequent value as base**: similar to the above, except that the data is analyzed and the most frequent value is used as a base. So, if most examples have the value middle, the two newly constructed continuous attributes will be status=low and status=high.
-   - **One attribute per value**: this would construct three continuous attributes out of a three-valued discrete one.
-   - **Ignore multinominal attributes**: removes the multinominal attributes from the data.
-   - **Treat as ordinal**: converts the attribute into a continuous attribute with values 0, 1, and 2.
-   - **Divide by number of values**: same as above, except that the values are normalized into range 0-1. So, our case would give values 0, 0.5 and 1.
+    Examples in this section will assume that we have a discrete attribute status with the values low, middle and high, listed in that order. Options for their transformation are:
+
+   - **First value as base**: a N-valued categorical variable will be transformed into N-1 numeric variables, each serving as an indicator for one of original values except for the base value. The base value is the first value in the list. By default, the values are ordered alphabetically; their order can be changed in [Edit Domain](../data/editdomain).
```

Comment:
an indicator for one of *the* original values

I think.

Reply:
I wrote *the*, but wasn't sure. :)